packages/sqs-receiver-service/src/receiver.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+import db from 'debug'
+import Bottleneck from 'bottleneck'
+import environment from './environment.js'
+import readQueue from './read-queue.js'
+import processMessage from './process-message.js'
+import deleteMessages from './delete-messages.js'
+import showQueueStatistics from './show-queue-statistics.js'
+
+/**
+ * This is the process for a single SQS queue
+ */
+const debug = db('receiver')
+
+// Validate the environment and return a standard object
+const { env } = environment(process.env, process.env.RECEIVER_PREFIX)
+
+console.log(`Running receiver process:${JSON.stringify(env, null, 4)}`)
+
+const limiter = new Bottleneck({
+  minTime: env.SUBSCRIBER_RATE_LIMIT_MS,
+  maxConcurrent: env.SUBSCRIBER_PARALLEL_LIMIT
+})
+
+const processMessageLimited = limiter.wrap(processMessage)
+
+/**
+ * An infinite async loop to poll the queue
+ * @returns {Promise<void>}
+ */
+const receiver = async () => {
+  // Read the SQS message queue
+  const messages = await readQueue(env.URL, Number.parseInt(env.VISIBILITY_TIMEOUT_MS), Number.parseInt(env.WAIT_TIME_MS))
+
+  // If we have read any messages then post the body to the subscriber
+  if (messages) {
+    debug(`Read ${messages.length} messages...`)
```

Comment:
You can use a format string with debug and pass messages as an argument rather than calling debug twice:

debug(`Read ${messages.length} messages...%O`, {messages})